esptool.py

```diff
@@ -18,6 +18,8 @@
 # this program; if not, write to the Free Software Foundation, Inc., 51 Franklin
 # Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
+from __future__ import print_function, unicode_literals, division
+
```

Comment:
added to the flake8 config. As for whether to use unicode_literals or not, I will defer to the big boss @themadinventor as to what he thinks